packages/render-fidelity-tools/src/common.ts

```diff
@@ -174,80 +168,79 @@ export class ImageComparator {
             colorDelta(candidateImage, goldenImage, position, position);
         const exactlyMatched = (delta <= thresholdSquared ? 1 : 0) * 255;
 
-        if (exactlyMatched) {
-          matched++;
-        } else {
-          mismatchingSum += delta;
-        }
-
         const thresholdDelta = Math.max(0, delta - thresholdSquared);
 
-        sum += thresholdDelta;
-        squareSum += delta * delta;
-
-        if (generateVisuals) {
-          const deltaIntensity =
-              Math.round(255 * thresholdDelta / MAX_COLOR_DISTANCE);
-
-          maximumDeltaIntensity =
-              Math.max(deltaIntensity, maximumDeltaIntensity);
-
-          this.drawPixel(
-              blackWhiteImage!,
-              position,
-              exactlyMatched,
-              exactlyMatched,
-              exactlyMatched);
-          this.drawPixel(
-              deltaImage!,
-              position,
-              255,
-              255 - deltaIntensity,
-              255 - deltaIntensity);
-        }
+        const deltaIntensity =
+            Math.round(255 * thresholdDelta / MAX_COLOR_DISTANCE);
+
+        maximumDeltaIntensity = Math.max(deltaIntensity, maximumDeltaIntensity);
+
+        this.drawPixel(
+            blackWhiteImage!,
+            position,
+            exactlyMatched,
+            exactlyMatched,
+            exactlyMatched);
+        this.drawPixel(
+            deltaImage!,
```

Comment:
Are all the bangs after `deltaImage` still necessary, now that it is created unconditionally?